acceptance/ui/features/steps/hosts_steps.rb

```diff
@@ -15,9 +27,38 @@
 end
 
 When /^I click the Add-Host button$/ do
-    clickAddHostsButton()
+    clickAddHostButton()
+end
+
+Then (/^the "Active" column should be sorted with active hosts on (top|the bottom)$/) do |order|
+  list = page.all("[ng-if$='host.active']")
```

Comment:
The selector matches active and inactive hosts, but I haven't tested any bad (red exclamation mark) hosts.